src/components/ArticleCard.vue

```diff
@@ -1,231 +1,169 @@
-    async handleClickBookmark() {
-      try {
-        if (!this.isSubscribed) {
-          await this.$store.dispatch('article/SUBSCRIBE', this.articleId)
-        } else {
-          this.$store.dispatch('article/UNSUBSCRIBE', this.articleId)
-        }
-      } catch (error) {
-        if (!this.isLogin) {
-          this.$bvModal.msgBoxOk(`尚未登入！追蹤文章失敗`, {
-            title: '追蹤文章',
-            okVariant: 'danger',
-            okTitle: '確定',
-            footerClass: 'modal-footer-confirm',
-            centered: true
-          })
-        } else {
-          this.$bvModal.msgBoxOk(
-            `${this.isSubscribed ? '取消' : ''}追蹤文章失敗`,
-            {
-              title: '追蹤文章',
-              okVariant: 'danger',
-              okTitle: '確定',
-              footerClass: 'modal-footer-confirm',
-              centered: true
-            }
-          )
-        }
-      }
-    },
-    getArticleContent: Utils.getArticleContent
+    GotoArticle() {
+      this.$router.push(`article/${this.articleId}`)
+    }
   }
 }
 </script>
 
-<style lang="scss" scoped>
-@import '@/assets/scss/news.scss';
+<style lang="scss" >
+.article-card {
+  position: relative;
+  flex: 1;
+  width: 320px;
+  max-height: 388px;
+  min-width: 304px;
+  border-radius: 16px;
+  box-shadow: 0 4px 15px 0 rgba(0, 0, 0, 0.08);
 
-.card {
-  padding: 15.2px;
-  max-width: 320px;
-  height: 380px;
-  width: 100%;
+  &:hover {
+    box-shadow: 0 4px 30px 0 rgba(0, 0, 0, 0.16);
+    cursor: pointer;
+  }
 
-  &--wide {
-    max-width: 800px;
-    height: 276px;
+  &__body {
+    padding: 1.25rem;
+    padding-bottom: 1.5rem;
   }
 
-}
-.heading {
-  // font-size: 22px;
-  line-height: 1.5em;
-  max-height: 3em;
-  overflow: hidden;
-  h4 {
-    margin: 0;
+  &__image {
+    height: 220px;
+    overflow: hidden;
+    border-radius: 16px 16px 0 0;
+    margin: auto;
+
+    img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
   }
-  box-sizing: border-box;
-  text-align: justify;
-  margin-top: 26px;
-  margin-bottom: 29px;
-}
 
-.article-excerpt {
-  line-height: 1.5em;
-  overflow: hidden;
-  margin-bottom: 20px;
-  display: -webkit-box;
-  -webkit-line-clamp: 7;
-  -webkit-box-orient: vertical;
-  &--wide {
-    -webkit-line-clamp: 4;
+  &__category {
+    font-size: 0.875rem;
+    color: $gray-8;
+    line-height: 1.25rem;
+    -webkit-line-clamp: 2;
+  }
+
+  &__title {
+    font-size: 1.125rem;
+    font-weight: 600;
+    line-height: 1.875rem;
+    color: $text-1;
+
+    display: -webkit-box;
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 2;
+    overflow: hidden;
+  }
+
+  &__hashtag {
+    display: flex;
+    flex-direction: row;
+    gap: 0.5rem;
+    margin-top: 0.5rem;
+  }
+
+  &__badge {
+    position: absolute;
+    top: 20px;
+    left: 20px;
   }
-}
 
-.subscribe-btn {
-  position: absolute;
-  top: 1rem;
-  right: 1rem;
-  background: transparent;
-  border: 0;
-  padding: 0;
-  margin: 0;
+  &__lg {
+    width: 392px;
```

Comment:
Which design does the `__lg` class correspond to? Or does this large attribute correspond to the profile page card?

Reply:
Corresponding to the large card design in large viewport.